.github/workflows/pants.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# See https://pants.readme.io/docs/using-pants-in-ci for tips on how to set up your CI with Pants.
+
+name: Pants
+
+on: [push, pull_request]
+
+jobs:
+  org-check:
+    name: Check GitHub Organization
+    if: ${{ github.repository_owner == 'pantsbuild' }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Noop
+        run: "true"
+  build:
+    name: Perform CI Checks
+    needs: org-check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      id: cache
+      with:
+        path: |
+          ~/.cache/pants/setup
+          ~/.cache/pants/lmdb_store
+          ~/.cache/pants/named_caches
+        key: ${{ runner.os }}-
+    - name: Bootstrap Pants
+      run: ./pants --version
+    - name: Check Pants config files
+      run: |
+        ./pants tailor --check  # Check if any targets missing
+        ./pants update-build-files --check  # Check for deprecations and formatting
```

Comment:
Can do both of these in one invoke?

Reply:
Yeah. Looking at this makes me frustrated there are two separate commands. Alas

Reply:
Soon ™️